airflow/cli/cli_parser.py

```diff
@@ -277,6 +277,15 @@ def _check(value):
     ),
     action="store_true",
 )
+ARG_RERUN_SUCCEED_TASKS = Arg(
+    ("--rerun-succeeded-tasks",),
+    help=(
+        "if set, the backfill will auto-rerun "
+        "all the succeeded tasks for the backfill date range "
+        "instead of throwing exceptions"
```

Comment:
Is this likely to throw exceptions if not set?

Reply:
if not set, it won't throw exceptions, the program will set rerun_succeeded_tasks to false by default.

Reply:
So the part you said `instead of throwing exceptions should be removed.

Reply:
This is written in imitation of the rerun_failed_tasks function

Reply:
and i can remove it, thank you